### ISSUE TITLE
bake fails when version field is omitted from compose file

### ISSUE THREAD

**Issue by dudicoco** [docker/buildx#719]
When the `version` field is omitted from a compose file (as per the new specification), the `bake` command fails.

Example file:
```yaml
services:
  app:
    image: test:latest
    build: .
```

Output:
```sh
$ docker buildx bake
[+] Building 0.0s (0/0)
error: unsupported Compose file version: 1.0
```

**Comment by crazy-max** [comment]
@dudicoco Should have been fixed since buildx 0.6.0. What's the output of `docker buildx version`?

**Comment by dudicoco** [comment]
Solved by upgrading buildx version.
